modules/tinymce/src/core/main/ts/api/NotificationManager.ts

```diff
@@ -163,11 +169,13 @@ const NotificationManager = (editor: Editor): NotificationManager => {
       Delay.requestAnimationFrame(reposition);
     });
 
-    editor.on('remove', () => {
-      Arr.each(notifications.slice(), (notification) => {
-        getImplementation().close(notification);
-      });
+    editor.on('AfterProgressState', (evt) => {
```

Comment:
As discussed, this logic shouldn't be here since it's not classified as a "popup". So we want notifications to stay when the throbber activates.